maintainer/cuda_build.sh

```diff
@@ -0,0 +1,31 @@
+GIT_COMMIT=$(git rev-parse HEAD)
+URL=$(echo "https://gitlab.icp.uni-stuttgart.de/kai/espresso-build/-/jobs/${CI_JOB_ID}")
+STATUS="pending"
+curl "https://api.github.com/repos/kaiszuttor/espresso/statuses/$GIT_COMMIT?access_token=$GITHUB_TOKEN" \
+      -H "Content-Type: application/json" \
+      -X POST \
+      -d "{\"state\": \"$STATUS\", \"context\": \"ICP CUDA build\", \"target_url\": \"$URL\"}"
```

Comment:
I dont want to discuss on details such as names at this early stage. Actually the purpose of the test is running CUDA code. The status is set by the build stage, a failing doc will not cause a red cross.